docs/development/key_generation.md

```diff
@@ -0,0 +1,23 @@
+### Key generation
+
+Keys are generated using [subkey](https://substrate.dev/docs/en/knowledgebase/integrate/subkey) or use `tfchain key`.
```

Comment:
Can we elaborate on this one, I don't think I saw the key role in the other files in the development directory